src/utils/store.js

```diff
@@ -0,0 +1,19 @@
+import { applyMiddleware, createStore } from 'redux';
+import { composeWithDevTools } from 'redux-devtools-extension';
+import thunk from 'redux-thunk';
+import { createLogger } from 'redux-logger';
+import reducers from '../reducers/index';
+
+const middleWare = [];
+middleWare.push(thunk);
+
+const loggerMiddleware = createLogger({
+  predicate: () => process.env.NODE_ENV === 'development',
+});
+
+middleWare.push(loggerMiddleware);
+
+export default createStore(
+  reducers,
+  composeWithDevTools(applyMiddleware(...middleWare))
```

Comment:
I think it would be nice to do a check to ensure that we are using redux dev tools only in _development_ and not in production @McHardex

Reply:
Alright. I will check out how I can make that happen

Reply:
Fixed this. Came from my initial html webpack plugin setup